api/lib/request/request.ts

```diff
@@ -12,21 +12,24 @@ const request = require('request-promise')
 module.exports = (method, url, params) => {
     const headers = params.headers && config.configEnv !== 'mock' ? Object.assign(params.headers) : {}
 
-    let options = {
+    const options = {
         body: '',
         formData: '',
-        method,
-        url,
         headers: {
             ...headers,
-            'Content-Type': params.headers['Content-Type'] || 'application/json'
-
+            'Content-Type': params.headers['Content-Type'] || 'application/json',
         },
-        json: true
+        json: params.json !== undefined ? params.json : true,
```

Comment:
this will pass null also
  json: params.json || true,
  maybe thats a style thing mostly

Reply:
good call, I remember I had to pass the param, otherwise it was being sent & returned as json as opposed to multipart/form-data. In fact, this probably won't be needed as much once we convert the upload document part to use axios etc.

Reply:
No longer using, as promise-request split out into separate file uploadFileFormData.ts